### PR TITLE
Add ovirt_networks and ovirt_networks_facts modules

### PR DESCRIPTION
**Pull request by machacekondra**
##### ISSUE TYPE

- New Module Pull Request
##### COMPONENT NAME

- ovirt_networks 
- ovirt_networks_facts
##### ANSIBLE VERSION

```
ansible 2.1.1.0
```
##### SUMMARY

This PR adds ovirt_networks and ovirt_networks_facts modules to manage oVirt logical networks.

```

```

### CODE REVIEW

**Comment by machacekondra** [comment]
@mwperina @ryansb

**Comment by machacekondra** [comment]
ready_for_review

**Comment by gregdek** [comment]
Thanks @machacekondra for this new module. When this module receives 'shipit' comments from two community members and any 'needs_revision' comments have been resolved, we will mark for inclusion.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by machacekondra** [comment]
shipit

**Comment by gregdek** [comment]
Thanks @machacekondra. Since you are one of the maintainers (@machacekondra) of this module and you commented with "shipit", we are marking this PR for inclusion.

[This message brought to you by your friendly Ansibull-bot.]

File: cloud/ovirt/ovirt_networks.py [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+extends_documentation_fragment: ovirt
+'''
+
+EXAMPLES = '''
+# Examples don't contain auth parameter for simplicity,
+# look at ovirt_auth module to see how to reuse authentication:
+
+# Create network
+- ovirt_networks:
+    auth: "{{ ovirt_auth }}"
```
Comment (mwperina) [inline cloud/ovirt/ovirt_networks.py]: Please remove auth parameter to be consistent with other modules and examples note

File: cloud/ovirt/ovirt_networks_facts.py [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+        - "For example to search network starting with string vlan1 use: name=vlan1*"
+extends_documentation_fragment: ovirt
+'''
+
+
+EXAMPLES = '''
+# Examples don't contain auth parameter for simplicity,
+# look at ovirt_auth module to see how to reuse authentication:
+
+# Gather facts about all networks named C<vlan1*>:
```
Comment (mwperina) [inline cloud/ovirt/ovirt_networks_facts.py]: I'd rephrase to 'Gather facts about all networks which names start with C<vlan1>:

**Review by mwperina — CHANGES_REQUESTED** [review]

File: cloud/ovirt/ovirt_networks_facts.py [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+        - "For example to search network starting with string vlan1 use: name=vlan1*"
+extends_documentation_fragment: ovirt
+'''
+
+
+EXAMPLES = '''
+# Examples don't contain auth parameter for simplicity,
+# look at ovirt_auth module to see how to reuse authentication:
+
+# Gather facts about all networks which names start with C(vlan1*):
```
Comment (mwperina) [inline cloud/ovirt/ovirt_networks_facts.py]: 'C(vlan1*)' -> 'C(vlan1)'

**Review by mwperina — CHANGES_REQUESTED** [review]

**Review by mwperina — APPROVED** [review]
Looks good to me, thanks! :+1:

**Comment by bcoca** [comment]
so most of the  modules seem to have same issues as on the ticket i've already commented on, I'm going to remove shipit from all of them until the first ticket gets through.

FYI, we normally require an initial ticket for 1 module as it allows us to debug any systemic issues before you submit the rest.

**Comment by gregdek** [comment]
Thanks again to @machacekondra for this PR. This PR was reviewed by an Ansible member. Marking for inclusion.

However, we would still appreciate if the maintainer(s) @machacekondra could look over it.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by machacekondra** [comment]
Will fix. There are already three modules merged:

 https://github.com/ansible/ansible-modules-extras/pull/2836

**Comment by machacekondra** [comment]
shipit

**Review by mwperina — APPROVED** [review]
:+1:
